src/blocks.rs

```diff
@@ -76,6 +76,24 @@ use crate::input::I3BarEvent;
 use crate::scheduler::Task;
 use crate::widget::I3BarWidget;
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum Refresh {
```

Comment:
If we go with this then perhaps Update might be better, as you mentioned.

Reply:
I'll rename `Refresh` to `Update` then =)